src/app/store/auth/auth.reducer.ts

```diff
@@ -5,12 +5,18 @@ import * as AuthActions from './auth.actions';
 export const authFeatureKey = 'authState';
 
 export interface AuthStateInterface {
-  pending: Pending;
+  pending: {
```

Comment:
why did you create separated fields for register and login pending?
can we use only one instead?